### DISCUSSION TITLE
permission denied

### DISCUSSION THREAD

**Discussion by safayavatsal** [community/community#17746] [GitHub Education] [answered]
remote: Permission to education/GitHubGraduation-2022.git denied to safayavatsal.
fatal: unable to access 'https://github.com/education/GitHubGraduation-2022.git/': The requested URL returned error: 403

**Comment by hendramanudinata03** [comment, marked as answer]
You need to fork the repository to your account. Then, push your commit into your forked repository and submit a PR.
